### ISSUE TITLE
Unexpected Outputs

### ISSUE THREAD

**Issue by ruveydayilmaz0** [Imraj-Singh/Score-Based-Generative-Models-for-PET-Image-Reconstruction#6]
Hi @Imraj-Singh ,

I tried training the model on Brainweb dataset and generating 2D data using the DDS sampler with the parameter values given in the paper. I get unexpected outputs as shown below. Have you encountered this before, and do you have a suggestion for correcting this problem?

<img width="400" height="380" alt="Image" src="https://github.com/user-attachments/assets/544f57b8-9109-461c-8495-2ad7f1954bee" />

**Comment by Imraj-Singh** [comment]
Have you tried getting an unconditional sample from your trained score-model. Does that look like a brain?

It looks like you may have too much data consistency and you're over fitting to noise in the measurements. Did you use the normalisation procedure?

**Comment by ruveydayilmaz0** [comment]
> Have you tried getting an unconditional sample from your trained score-model. Does that look like a brain?
> 
> It looks like you may have too much data consistency and you're over fitting to noise in the measurements. Did you use the normalisation procedure?

I checked the output of the score-model without measurement adaptation, it looks reasonable (similar to training data).

I use the estimate_scale_factor() function in inference, which is supposed to normalize the data accordingly. I use the default DDS config to generate data with anchor.

**Comment by Imraj-Singh** [comment]
FYI, for 2D there are two noise levels with separate number of epochs for project and beta values. You could try increasing the amount of anchoring and seeing if you approach the unconditional sample (i.e. effectively remove the measurement adaptation). Then try introduce the measurement adaptation back in.
